app/js/deleteQuestion.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * soft deletes question from database
```

Comment:
not accurate. the API controls what this does. all the Js knows is it deletes the record. if the API changes, this comment will be wrong, but no1 will update it. dont write comments about code that isnt the code your writting.